// 创建房间
message CSCreateRoomRequest
{
}
message CSCreateRoomResponse
{
    optional int32 result               = 1;
}

// 请求加入房间
message CSJoinRoomRequest
{
    optional int32 uin                  = 1;
}
message CSJoinRoomResponse
{
    optional int32 result               = 1;
}

//请求开始比赛，已经准备好
message CSFightReadyRequest
{
}
message CSFightReadyResponse
{
    optional int32 result               = 1;
}